### PR TITLE
Experiment code owners feature

### PR DESCRIPTION
**Pull request by brunoborges**
This may help us better organize who owns what :-)

More information: https://github.com/blog/2392-introducing-code-owners

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+*                         @brunoborges @Djelibeybi
+
+OracleDatabase/*          @gvenzl
+OracleCoherence/*         @thegridman
+OracleJava/*              @aureliogrb
+OracleWebLogic/*          @mriccell
+OracleHTTPServer/*        @prkishor
+OracleInstantClient/*     @gvenzl
+OracleFMWInfrastructure/* @gvenzl
```
Comment (gvenzl) [inline CODEOWNERS]: I'm not the code owner for FMWInfrastucture. :)

Comment (Djelibeybi) [inline CODEOWNERS]: You are now. Congratulations! :)

Comment (brunoborges) [inline CODEOWNERS]: LOL

**Comment by brunoborges** [comment]
@gvenzl @mriccell ping? :P

**Comment by gvenzl** [comment]
Pong

**Pull request merged**